core/state/statedb.go

```diff
@@ -968,7 +1015,11 @@ func (s *StateDB) IntermediateRoot(deleteEmptyObjects bool) common.Hash {
 	// to pull useful data from disk.
 	for addr := range s.stateObjectsPending {
 		if obj := s.stateObjects[addr]; !obj.deleted {
-			obj.updateRoot()
+			if s.trie.IsVerkle() {
+				obj.updateTrie()
```

Comment:
Does it mean in verkle, `obj.Root` is not relevant anymore? Because there is only 1 layer tree. If so should we just remove this field from the account(I know it will be a consensus change, just brainstorm it)?

Reply:
yes the account root no longer has a meaning